src/protocols/bitcoin/libs/BitcoinProtocol.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable class-methods-use-this */
+
+import type {
+  SpendTransactionFee,
+} from '@/types';
+import { BaseProtocol } from '@/protocols/BaseProtocol';
+import { PROTOCOL_BITCOIN } from '@/constants';
+
+type BitcoinAddress = `bc_${string}`;
+
+export class BitcoinProtocol extends BaseProtocol {
+  static title = PROTOCOL_BITCOIN;
+
+  static symbol = '₿';
+
+  static derivationPath = 'm/44h/0h';
+
+  async spend(amount: string, recipientId: BitcoinAddress, options: any): Promise<string> {
```

Comment:
Sender address/account would be provided in options?

Reply:
I am plannning to address to redinfe this function prototype when writing bitcoin adapter methods.